packages/components/src/internal/components/permissions/PermissionsRole.tsx

```diff
@@ -24,7 +24,6 @@ interface Props {
     principals: List<Principal>;
     role: SecurityRole;
     selectedUserId: number;
-    typeToShow: string;
```

Comment:
Seems as though we don't need this prop anymore, and probably won't in the future, so I took it out. If there's any reason we'd like to keep it around I can put it back in, though.

Reply:
I agree. This should no longer be needed if both LKB and LKSM are now supporting users and groups.